addons/website/static/src/js/content/menu.js

```diff
@@ -478,8 +478,8 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
         update: '_dropdownHover',
     }],
     events: {
-        'mouseenter .dropdown:not(.position-static)': '_onMouseEnter',
-        'mouseleave .dropdown:not(.position-static)': '_onMouseLeave',
```

Comment:
@qsm-odoo I guess ':not(.position-static)' was there for a good reason but I dont find it. The 'open on hover' works well on mega-menu.